stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -11,32 +11,32 @@
 //===----------------------------------------------------------------------===//
 
 #if DEPLOYMENT_RUNTIME_SWIFT
-
+    
```

Comment:
It is from me moving it to and from Xcode to test in swift-corelibs-foundation; ideally we should have a consistent story of copy-paste structural types to and from SCLF
So it is not a problem imho.